.github/workflows/appsignals-python-e2e-eks-test.yml

```diff
@@ -0,0 +1,393 @@
+        run: mkdir -p ${{ github.workspace }}/.kube
+
+      - name: Set KUBECONFIG environment variable
+        run: echo KUBECONFIG="${{ github.workspace }}/.kube/config" >> $GITHUB_ENV
+
+      - name: Set up kubeconfig
+        run: aws eks update-kubeconfig --name ${{ inputs.test-cluster-name }} --region ${{ inputs.aws-region }}
+
+      - name: Install eksctl
+        run: |
+          source ${{ env.TEST_RESOURCES_FOLDER }}/.github/workflows/util/execute_and_retry.sh
+          mkdir ${{ github.workspace }}/eksctl
+          curl -sLO "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_Linux_amd64.tar.gz"
+          execute_and_retry 2 "tar -xzf eksctl_Linux_amd64.tar.gz -C ${{ github.workspace }}/eksctl && rm eksctl_Linux_amd64.tar.gz"
+          echo "${{ github.workspace }}/eksctl" >> $GITHUB_PATH
+
+      - name: Create role for AWS access from the sample app
+        id: create_service_account
+        run: |
+          eksctl create iamserviceaccount \
+          --name service-account-${{ env.TESTING_ID }} \
+          --namespace ${{ env.PYTHON_SAMPLE_APP_NAMESPACE }} \
+          --cluster ${{ inputs.test-cluster-name }} \
+          --role-name eks-s3-access-${{ env.TESTING_ID }} \
+          --attach-policy-arn arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess \
+          --region ${{ inputs.aws-region }} \
+          --approve
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v3
+
+      - name: Checkout Amazon Cloudwatch Agent Operator
+        uses: actions/checkout@v4
+        with:
+          repository: aws/amazon-cloudwatch-agent-operator
+          # SHA as of March 4
+          ref: abf75babe672412cb63c56cbcf1c5ce2d8c97a1c
+          path: amazon-cloudwatch-agent-operator
+
+      - name: Edit Helm values for Amazon Cloudwatch Agent Operator
+        working-directory: amazon-cloudwatch-agent-operator/helm/
+        run: |
+          sed -i 's/clusterName:/clusterName: ${{ inputs.test-cluster-name }}/g' values.yaml
+          sed -i 's/region:/region: ${{ inputs.aws-region }}/g' values.yaml
+          sed -i 's/repository: cloudwatch-agent-operator/repository: cwagent-operator-pre-release/g' values.yaml
+          sed -i 's/tag: 1.0.2/tag: latest/g' values.yaml
+          sed -i 's/tag: 0.43b0/tag: latest/g' values.yaml
+          sed -i '0,/public: public.ecr.aws\/cloudwatch-agent/s//public: ${{ secrets.TEST_CW_AGENT_REPO }}/' values.yaml
+          if [ ${{ inputs.appsignals-adot-image }} != "" ]; then
+            echo "Using appsignals-adot-image"
+            sed -i 's~repository: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python~repository: ${{ inputs.appsignals-adot-image }}~g' values.yaml
+          else
+            echo "appsignals-adot-image is empty"
+            sed -i 's~repository: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python~repository: ${{ secrets.TEST_APP_SIGNALS_PYTHON_ADOT_IMAGE }}~g' values.yaml
+          fi
```

Comment:
This file from `aws/amazon-cloudwatch-agent-operator` is from commit hash `abf75babe672412cb63c56cbcf1c5ce2d8c97a1c`

Therefore I assume if it works once, it will always work. Since this instrumentation method is temporary, we won't need to use this helm file when we have a proper instrumentation method/script.